src/components/howTos.stories.mdx

```diff
@@ -0,0 +1,36 @@
+<Meta title="How-tos" />
+
+# How-to guides
+
+There are some particular use cases in this page. You can refer to the usage and some examples below.
+
+## Enabling content share in Electron applications
+
+If you are building a Electron app, you would need to create your own screen picker, and pass the `sourceId` to `useContentShareControls` hook to start content share.
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  useContentShareControls,
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => {
+  const { toggleContentShare } = useContentShareControls();
+
+  return (
+    <MeetingProvider>
+      <YourScreenPicker
+        onClickShareButton={async (selectedSourceId: string) => {
```

Comment:
We have JS SDK and component examples in JavaScript. Rewrite them into JavaScript.